csrankings-s.csv

```diff
@@ -1045,6 +1045,7 @@ Shubham Jain,Stony Brook University,https://www3.cs.stonybrook.edu/~jain,P9Z011Y
 Shubham Tulsiani,Carnegie Mellon University,https://shubhtuls.github.io,06rffEkAAAAJ
 Shubhangi K. Gawali,BITS Pilani-Goa,http://www.bits-pilani.ac.in/goa/shubhangi/profile,2hDkzdgAAAAJ
 Shubhangi Saraf,University of Toronto,https://www.math.toronto.edu/ssaraf,NiRqwkcAAAAJ
+Shubhra Kanti Karmaker Santu,https://karmake2.github.io,y6pZKT4AAAAJ

```

Comment:
No university listed.

Reply:
Also needs two entries, one per DBLP alias.

Reply:
I will make the necessary changes and create another pull request. Thanks for the feedback.